course_access_groups/models.py

```diff
@@ -30,11 +31,25 @@ class CourseAccessGroup(models.Model):
 
 class Membership(utils_models.TimeStampedModel):
     """
-    Student membership in a CourseAccessGroup.
+    Student membership in a Group.
     """
 
-    group = models.ForeignKey(CourseAccessGroup)
+    group = models.ForeignKey(Group)
     user = models.OneToOneField(
         get_user_model(),
         help_text='Student. A student can only be enrolled in a single Course Access Group.'
     )
+
+
+class Rule(utils_models.TimeStampedModel):
+    name = models.CharField(max_length=255, help_text='A description for this assignment rule.')
+    domain = models.CharField(max_length=255, db_index=True)
+    group = models.ForeignKey(Group)
+
+
+class CourseGroup(utils_models.TimeStampedModel):
```

Comment:
Now I've scrolled down and I see 'Group' and 'CourseGroup' How about adding a README that explains what the models are. My question now is why do we have a 'Group' model and a 'CourseGroup' model? What is the purpose of each?

Reply:
I've added a inline docstring for now. I'll leave the README documentations after we finalize the feature and are sure that it actually works as expected.

There's a couple of unknowns to this point on how it would actually work, so documentation is not really possible.